setup.py

```diff
@@ -2,81 +2,91 @@
 # -*- coding: utf-8 -*-
 import os
 
-from setuptools import (
-    setup,
-    find_packages,
-)
+from setuptools.command.develop import develop as _develop
+from setuptools import setup, find_packages
+from subprocess import check_call
 
 
 DIR = os.path.dirname(os.path.abspath(__file__))
+readme = open(os.path.join(DIR, "README.md")).read()
+WEB3_VERSION = "5.0.0b2,<6.0.0"
+
+
+class develop(_develop):
+    """
+    Post-installation to make sure web3 tester extras are installed.
+    """
 
+    def run(self):
+        _develop.run(self)
+        check_call(["pip", "install", f"web3[tester]>={WEB3_VERSION}"])
```

Comment:
I don't think this will work when it's installed via a wheel (but I'm not positive) you should confirm before proceeding.  Generally this is in the realm of things that I consider bad ideas that don't end up reliably working beyond the development environment.

Reply:
I agree it's a little iffy, but isn't this a command that will *only* be run in a development/testing environment?

Reply:
> I don't think this will work when it's installed via a wheel (but I'm not positive) 
This is true from what I've read.

> isn't this a command that will only be run in a development/testing environment?
I feel the same way, but I guess it's possible other projects might have a core dependency on `ethpm[dev]` - which would be weird, but possible. IMO though, this is the cleanest solution I've come across for removing the `web3[tester]` splinter from ethpm's core dependencies - and worth the trade-off, but I could be misjudging this.